### DISCUSSION TITLE
I would like to checkout users to licenses via API/Script - Is this possible ?

### DISCUSSION THREAD

**Discussion by denisimmoos** [snipe/snipe-it#10326] [Q&A] [answered]
Hi,

I would like to add all our office365 licenses to snipe-it with an automation script.
I managed so far to get all licenses we own with the https://graph.windows.net API and it was possible to add them to snipe-it using  /api/v1/licenses.

Now I would like to checkout all the users we have to those licenses but there is no api/v1/licenses/{id}/checkout. 
Is this possible ?
I tried it also via a regular login but I fail to checkout users due to the parameter "_token".

**Comment by snipe** [comment, marked as answer]
https://snipe-it.readme.io/reference/licensesidseatseatid
